doc/api/crypto.md

```diff
@@ -4388,6 +4388,12 @@ An array of supported digest functions can be retrieved using
 <!-- YAML
 added: v0.11.14
 changes:
+  - version:
+      - v21.6.2
+      - v20.11.1
+      - v18.19.1
+    pr-url: https://github.com/nodejs-private/node-private/pull/525
```

Comment:
I think the commit metadata of https://github.com/nodejs/node/commit/54cd268059626800dbe1e02a88b28d9538cf5587 is wrong.

```suggestion
    pr-url: https://github.com/nodejs-private/node-private/pull/515
```

Reply:
>,< luckily noone but a few individuals can actually a) see the PR link and b) be able to access it. I only included the pr-url because without it docs wouldn't build.

And of course the 54cd268059626800dbe1e02a88b28d9538cf5587 metadata is the only thing I had to go by...